draft-ietf-ice-rfc5245bis.xml

```diff
@@ -4500,7 +4471,7 @@ is kept at a reasonable rate.
 </t>
 
 <t>
-The definition of a "reasonable" rate is that STUN should not use more
+The definition of a "reasonable" rate is that STUN MUST NOT use more
```

Comment:
I'm not sure we want this to be "MUST NOT" since in many cases it's impossible to know what bandwidth RTP will use because it adapts to network conditions.

Reply:
I can change it to "STUN is not supposed to use more..."